randovania/games/prime_hunters/layout/prime_hunters_configuration.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import dataclasses
+
+from randovania.bitpacking.bitpacking import BitPackDataclass
+from randovania.bitpacking.json_dataclass import JsonDataclass
+from randovania.game.game_enum import RandovaniaGame
+from randovania.games.prime_hunters.layout.force_field_configuration import ForceFieldConfiguration
+from randovania.layout.base.base_configuration import BaseConfiguration
+
+
+@dataclasses.dataclass(frozen=True)
+class HuntersOctolithConfig(BitPackDataclass, JsonDataclass):
+    prefer_bosses: bool
+    # required_octoliths: int = dataclasses.field(metadata={"min": 0, "max": 8, "precision": 1})
```

Comment:
?

Reply:
I added this for the future, but i guess that doesn't make sense to include for now.